### PR TITLE
Release 1.7.0

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

File: foglamp.version [hunk 1]
```diff
@@ -1 +1 @@
-foglamp_version>=1.6
+foglamp_version>=1.7
```
Comment (praveen-garg) [inline foglamp.version:1]: @Singhal-Vaibhav Missing service_notification.version. Please add.

Here we have foglamp.version as well, is this really required @pintomax @MarkRiddoch
I think we are NOT keeping foglamp.version file in rule- plugins; as direct dependency is notification service which has dependency check for foglamp version

**Review by Singhal-Vaibhav — DISMISSED** [review]

File: foglamp.version (hunk 1 above)
Comment (pintomax) [reply foglamp.version:1]: @praveen-garg we need both foglamp.version and  service_notification.version.

foglamp.version is being used by make_rpm and make_deb, any plugin.
service_notification.version is taken only for notification-rule and notification-delivery plugins

**Review by pintomax — DISMISSED** [review]
We have 1.7 in VERSION, foglamp.version and service_notification_version = OK

File: foglamp.version (hunk 1 above)
Comment (MarkRiddoch) [reply foglamp.version:1]: service_notification.version should certainly be added. As for having foglamp_version, it does no harm, and it does allow for the plugin to depend on a feature in FogLAMP that the service does not use.

**Review by MarkRiddoch — CHANGES_REQUESTED** [review]
We should add the dependency on the version of the notification server. This goes for all notification rule and delivery plugins.

**Review by MarkRiddoch — DISMISSED** [review]

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**